.github/workflows/main.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.30
-          args: --timeout 5m
+          args: --timeout 5m --skip-dirs pkg/gen
```

Comment:
also pkg/apis?

Reply:
sure

Reply:
I think you are right. deepcopy code is generated. We can ignore that file.